crowbar_framework/app/models/backup.rb

```diff
@@ -75,14 +75,23 @@ def restore(options = {})
     from_upgrade = options.fetch(:from_upgrade, false)
 
     if Crowbar::Backup::Restore.restore_steps_path.exist?
+      logger.debug I18n.t("backups.index.multiple_restore")
```

Comment:
We usually don't put localized messages in the logs (to avoid having to read french messages ;-))